packages/tailwindcss-colors/0.1.0/_manifest.yml

```diff
@@ -0,0 +1,6 @@
+name: "tailwindcss-colors"
+title: "Tailwind CSS Colors"
+description: "A package that provides TailwindCSS's default color palette as hex color codes for quick access."
+version: 0.1.0
+author: "Paulina Kalicka"
+tags: ["colors", "hex", "tailwindcss"]
```

Comment:
Would you kindly add a `homepage:` tag and value, please, even if the latter is simply `https://github.com/paulinek13/.fork.espanso-hub/tree/main`.
We've had problems fully merging packages where a `homepage:` is missing.